server/cloudfront-gen-lambda.js

```diff
@@ -448,6 +449,29 @@ if (blipConfig.COOKIE_BANNER_CLIENT_ID !== 'disabled') {
   indexHtml = indexHtml.replace(reCookieBanner, '$1  <!-- disabled -->\n$3')
 }
 
+/*Add hash for palette.css file*/
+let paletteDir = path.resolve(`/dist/static`)
+fs.readdir(paletteDir, (err, files) => {
+  if (err) {
+    return console.error('Unable to scan directory /dist/static: ' + err)
+  }
+
+  // Filter and find the file starting with 'palette'
+  // Regular expression to match palette.<md5>.css
+  const md5Pattern = /^palette\.[\da-f]{20}\.css$/i
+
+  // Find the file matching the pattern
+  let md5File = files.find(file => md5Pattern.test(file))
+
+  if(md5File) {
+    console.log('Palette md5 file found, serving md5 file in index.html')
+    let rePalette = /(.*)(palette\.css)(.*)/m
```

Comment:
what about using a strict value here? to avoid using a reegex?

Reply:
you're right, i'm changing it